en/docs/install-and-setup/setup/deployment-best-practices/performance-and-capacity-planning.md

```diff
@@ -0,0 +1,260 @@
+
+#### Throughput comparison:
+
+A throughput increase is observed in the transformation scenario in API-M 3.2.0, in comparison to API-M 3.1.0
+
+The throughput increases up to a certain limit when the number of concurrent users increases. The Mediation API throughput increase rate is much lower than the Echo API. Throughput decreases when the message sizes increase. Throughput decreases when the backend sleep time increase. This observation is similar to both APIs. This means that if the backend takes more time, the request processing rate at the API Manager Gateway will be less.
+
+#### Key observations related to response time:
+
+The average response time increases when the number of concurrent users increases. The increasing rate of average response time for both API-M 3.2.0 and API-M 3.1.0 is similar.
+
+The average response time increases when the number of concurrent users increases. The average response time increases considerably for Mediation API when the message sizes increase due to the message processing. The average response time of the Echo API does not increase as much as the Mediation API. The average response time increases when the backend sleep time increases. This observation is similar to both APIs.
+
+#### Key observations related to GC throughput:
+
+The GC throughput decreases when the number of concurrent users increases. When there are more concurrent users, the object allocation rate increases. The GC throughput increases when the message size increases. The request processing rate slows down due to the time taken to process large messages. Therefore, the object allocation rate decreases  when the message size increases. The GC throughput increases when the backend sleep time increases. The object allocation rate will be low when the backend takes more time to respond.
+
+## Comparison of 3.1.0 and 3.2.0
+
+### Average response time comparison
+
+#### Average response time vs concurrent users
+
+  [![]({{base_path}}/assets/img/setup-and-install/performance-test-results/average_time_0ms_50B.png)]({{base_path}}/assets/img/setup-and-install/performance-test-results/average_time_0ms_50B.png)
+  [![]({{base_path}}/assets/img/setup-and-install/performance-test-results/average_time_0ms_1KiB.png)]({{base_path}}/assets/img/setup-and-install/performance-test-results/average_time_0ms_1KiB.png)
+  [![]({{base_path}}/assets/img/setup-and-install/performance-test-results/average_time_0ms_10KiB.png)]({{base_path}}/assets/img/setup-and-install/performance-test-results/average_time_0ms_10KiB.png)
+
+### GC Throughput comparison 
+
+ [![]({{base_path}}/assets/img/setup-and-install/performance-test-results/gc_throughput_0ms_1KiB.png)]({{base_path}}/assets/img/setup-and-install/performance-test-results/gc_throughput_0ms_1KiB.png)
```

Comment:
The file names of the images should not have uppercase letters in it, because based on the current mkdocs setup such images will not appear. Therefore, the actual image names and the references to them in this file need to be changed.